specifyweb/frontend/js_src/lib/components/Atoms/Internationalization.ts

```diff
@@ -153,8 +153,7 @@ export function getRelativeDate(date: Readonly<Date>): string {
      * create a data set and see its date of creation be 5 hours into the
      * future
      */
-    // Throw new Error('Future dates are not supported');
-    console.error('Future dates are not supported');
+    // Throw new Error('Future dates are not supported'), removed console.log in issue #4051;
```

Comment:
(very optional)

```suggestion
    // Throw new Error('Future dates are not supported'), removed console.log in #4071;
```

or if #4070 is being used as the PR for fixing #4051:

```suggestion
    // Throw new Error('Future dates are not supported'), removed console.log in #4051;
```

It may be **_very slightly_** more helpful for future developers to have a link to the Pull Request over the issue. The Pull Request generally should include more information which may be immediately more helpful such as code changes, what was causing the problem, testing instructions, potential problems with code changes, etc. 
It _may_ save future developers a click if they do not need the issue for context (which is arguably rare)